bin/setup_stack

```diff
@@ -178,7 +184,7 @@ fi
 
 echo Connecting faux interfaces...
 add_faux t2sw1 1 1 $faux_dhcp
-add_faux t2sw2 1 2 $faux_dhcp
+add_faux t2sw2 1 2 $faux_dhcp $faux_http
```

Comment:
can this just be fixed?  Does anything bad happen if we just always enable http on this device?  I'm trying to keep the overall complexity down unless there's a specific reason to make it configurable!

Reply:
Done. The http server is always running on faux-2 now.